client/my-sites/domains/components/domain-and-plan-package/navigation/index.tsx

```diff
@@ -26,7 +29,7 @@ export default function DomainAndPlanPackageNavigation( props ) {
 	return (
 		<div className="domain-and-plan-package-navigation">
 			<div className="domain-and-plan-package-navigation__back">
-				<Button borderless="true" onClick={ goBack }>
+				<Button borderless onClick={ goBack }>
```

Comment:
After converting the file, TS flagged this boolean-as-string. After fixing, the linter helpfully pointed out that the value should be omitted. I've tested locally to confirm the button style isn't impacted by this change.